File: src/leetcode/leetcode2490/Solution.java
package leetcode.leetcode2490;
//2490. Circular Sentence
// https://leetcode.cn/problems/circular-sentence/
class Solution {
    public boolean isCircularSentence(String sentence) {
        String[] arr = sentence.split(" ");
        if(sentence.charAt(0)!=sentence.charAt(sentence.length()-1)) return false;
        for (int i = 0; i < arr.length-1; i++) {
            if(arr[i].charAt(arr[i].length()-1)!=arr[i+1].charAt(0)) return false;
        }
        return true;
    }
    public static void main(String[] args) {
        System.out.println(new Solution().isCircularSentence("LOl lal lsa asL"));
        
    }
}

File: src/leetcode/leetcode2490/Solution.py
class Solution:
    def isCircularSentence(self, sentence: str) -> bool:
        if sentence[len(sentence)-1] != sentence[0]:
            return False
        arr=sentence.split(' ');
        i=0
        while(i<len(arr)-1):
            if arr[i][len(arr[i])-1] != arr[i+1][0]:
                return False
            i=i+1
        return True


res = Solution().isCircularSentence("los sfg gll")
print(res)